indico/modules/events/abstracts/templates/display/submit_abstract.html

```diff
@@ -0,0 +1,28 @@
+{% from 'events/display/indico/_common.html' import render_users %}
+{% from 'forms/_form.html' import form_header, form_footer, form_rows, form_fieldset %}
+
+{% extends 'layout/conference_page_base.html' %}
+
+{% block title -%}
+    {% trans %}Submit abstract{% endtrans %}
+{%- endblock %}
+
+{%- block content %}
+    {% if cfa.is_open %}
```

Comment:
I'd move this part to a separate template (and render it before even doing form stuff in the RH if the CFA is not active). Right now you could circumvent the is_open condition by simply submitting the form with httpie, curl, etc since there is no check in the RH.